src/EventSourcingDb.Tests/ReadEventTypeTest.cs

```diff
@@ -20,7 +20,7 @@ public async Task FailsIfTheEventTypeDoesNotExist()
         }
         catch (HttpRequestException ex)
         {
-            Assert.Equal("Unexpected status code.", ex.Message);
+            Assert.StartsWith("Unexpected status code", ex.Message);
```

Comment:
The exception contains now the specific error from the server, this the change from `Equal` to `StartsWith`. I thin it is tedious to check the whole exception message, the important part is *that* the server error is contained.